develop/toolkit/parachains/fork-chains/chopsticks/get-started.md

```diff
@@ -13,7 +13,7 @@ This guide walks you through installing Chopsticks and provides information on c
 
 For additional support and information, please reach out through [GitHub Issues](https://github.com/AcalaNetwork/chopsticks/issues){target=_blank}.
 
-!!! note
+!!! tip
```

Comment:
IMO, this doesn't seem like a tip because it's highlighting a limitation of working with a Chopsticks forked network